### PR TITLE
Docs: Remove broken kubeval example.

### PR DESCRIPTION
**Pull request by etefera**
The example for the kubeval function doesn't point to any existing link. This PR deletes references to that example.

### CODE REVIEW

**Comment by frankfarzan** [comment]
Prachi has an inflight PR for adding kubeval, I'll defer to her on timing this.

**Review by prachirp — APPROVED** [review]
LGTM

**Comment by prachirp** [comment]
@frankfarzan @etefera The inflight PR will need an SDK change and release, a catalog functions version update and release, and updating the PR itself before it can be merged. I would rather remove this broken example and add it back later.

**Review by frankfarzan — APPROVED** [review]

**Pull request merged**
